### ISSUE TITLE
Doesn't work out of the box

### ISSUE THREAD

**Issue by Major-Mike** [iammukeshm/CustomUserManagement.MVC#7]
@iammukeshm I am not sure if you are checking these or planning on updating the repo to make it more current and useful, but thought since I spent some time with it, although disappointed that it doesn't work, wanted to let you know. Hopefully these are not out there just to get views and drive ad revenue and sincerely intended to be educational and useful.

Out of the box

![image](https://user-images.githubusercontent.com/55165479/219247918-1a6b8f2d-d5ed-44dc-8c99-185af3d450b7.png)

The project is also targeting 3.1 which is end of life, out of support and has security vulnerabilities. It would be helpful if you updated it for .NET 6 (LTS) or even .NET 7 (STS) as enough changes have been introduced to make the tutorial somewhat obsolete.

**Comment by iammukeshm** [comment]
Yes, I have noticed this. I will be writing new content for this and updating the repo..

**Comment by Major-Mike** [comment]
Awesome, good to hear. Look forward to seeing it.
